snyk/snyk
# feat: decrease to warning unsupported package manager error

when running with the --reachable flag on an unsupported
package manager, we would not stop the scan.

this is useful for having a global configuration that enables
this flag and want keep on testing with other package managers.


#### What does this PR do?
When using `snyk test --reachable` on a package manager that is not yet supported by ReachableVulns it used to stop running the cli and say Reachable Vulns is not supported... This has been changed to a warning message that displays at the bottom of the `snyk test` output.

#### Where should the reviewer start?
Notice that the unit test is gone and these is a new acceptance test. The exception is no longer thrown. Instead an alert is saved for later display to the user. In the acceptance test, the alert is looked for 

#### How should this be manually tested?
run `snyk test --reachability` on an npm project. See warning at the end of cli output.


#### Any background context you want to provide?
When one uses --reachable with snyk test, for a non-supported package manager, it used to throw an error and stop. This was a problem for automated CI when clients ran multiple projects through templated pipelines. This removes that stop and instead will run snyk test as normal and display a warning at the end that the particular package manager is not yet supported for reachability

#### What are the relevant tickets?
https://snyksec.atlassian.net/browse/FLOW-435

#### Screenshots
<img width="1346" alt="Screenshot 2020-10-05 at 20 32 16" src="https://user-images.githubusercontent.com/245346/95123576-18347f00-074a-11eb-99e1-14c021e21f99.png">


#### Additional questions
